transport/controller.cc

```diff
@@ -65,6 +66,124 @@ future<> controller::start_server() {
     return do_start_server().finally([this] { _ops_sem.signal(); });
 }
 
+// Wrapper for cql_server::listen to invoke it on all shards and log address and listen options.
```

Comment:
I personally am not a fan of short symbols with an extra explanation comments. For this reason I'd rename the function to something like `listen_on_all_shards`, which addresses the first part of the comment, and the 2nd part of the comment can be skipped - nobody ever explicitly mentions a need for logging something, we just log it. 
On a side note, comments with so many details tend to degrade quickly, better to include extra info in symbols' names.